kube-fluentd-operator/Dockerfile

```diff
@@ -1,20 +1,22 @@
 # builder image
 FROM golang:1.12 as builder
 
-ARG KFO_VERSION="v1.9.0"
+ARG KFO_VERSION="1.10.0"
+ARG FLUENTD_VERSION="1.7.3"
+
 WORKDIR /go/src/github.com/vmware/kube-fluentd-operator/config-reloader
 RUN go get -u github.com/golang/dep/cmd/dep
 COPY kube-fluentd-operator/config-reloader/. .
 
 # Speed up local builds where vendor is populated
 RUN [ -d vendor/github.com ] || make dep; true
 RUN make test
-RUN make build VERSION=$(KFO_VERSION)
+RUN make build VERSION=v1.10.0
```

Comment:
@cw-sakamoto 
Is this change correct?
I think i should use `KFO_VERSION`.

Reply:
@cw-ozaki 
That's fine😀, but I thought it would be easier in Dockerfile.tpl.

https://github.com/chatwork/dockerfiles/pull/132/files#diff-d85edd16bc18f6a82389b189877190daR14

Reply:
@cw-sakamoto 
That is, this command will no longer work.
```
docker build --build-args KFO_VERSION=xxx chatwork/ kube-fluentd-operator .
```
Specifying `ARG` for this Dockerfile does not change anything.

Reply:
@cw-ozaki Oh..! I see, you are right😃
I fixed build-args and unified version format in https://github.com/chatwork/dockerfiles/pull/132/commits/a37f8f523887de7d743686d8f55ca633be57ad33